src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -237,6 +246,50 @@ RestExecuteOnExtensionResponse handleRestExecuteOnExtensionRequest(RestExecuteOn
                 "No handler for " + ExtensionRestPathRegistry.restPathToString(request.getMethod(), request.getUri())
             );
         }
+
+        // TODO get this list of resource IDs and addition params from the RestExecuteOnExtentionRequest
```

Comment:
Tracked in https://github.com/opensearch-project/opensearch-sdk-java/issues/111 (should be completed by end of month)